scss/styles.scss

```diff
@@ -1,22 +1,22 @@
-// Libraries 
-@import "lib/bourbon/_bourbon.scss";
-@import "lib/neat/_neat.scss";
-@import "fonts";
+// Libraries
+@import 'lib/bourbon/_bourbon.scss';
+@import 'lib/neat/_neat.scss';
+@import 'fonts';
 
 // Variables and mixins
-@import "variables";
-@import "grid";
-@import "mixins/_utilities.scss";
-@import "mixins/_type-mixins.scss";
+@import 'variables';
+@import 'grid';
+@import 'mixins/_utilities.scss';
+@import 'mixins/_type-mixins.scss';
 
 // Base styling for html elements
-@import "elements/_elements.scss";
+@import 'elements/_elements.scss';
```

Comment:
Imported partial `elements/_elements.scss` should be written as `elements/elements`